DMTN-261.tex

```diff
@@ -1,54 +1,46 @@
 \setDocUpstreamLocation{\url{https://github.com/lsst-dm/dmtn-261}}
 
-\date{\vcsDate}
-
-% Optional: name of the document's curator
-% \setDocCurator{The Curator of this Document}
+\date{2024-06-21}
 
-\setDocAbstract{%
-This memo describes the agreement between Rubin/LSST and NOIRLab for the ANTARES broker to provide the LSST Alert Filtering Service.
-}
+\begin{document}
 
-% Change history defined here.
-% Order: oldest first.
-% Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
-% See LPM-51 for version number policy.
-\setDocChangeRecord{%
-  \addtohist{1}{YYYY-MM-DD}{Unreleased.}{Leanne Guy}
-}
+\setMemoto{ L. Guy, E. Bellm, P. Marshall}
+        
+\mkmemotitle
 
+During steady-state operations, Rubin Observatory's LSST will produce about ten million world public alerts of transients, variable, and moving objects. Science users will work with community alert
+brokers to crossmatch, filter, and classify these alerts in order to identify the subset that require real-time followup observations. 
+At the time the vision for this ambitious and comprehensive alert system was developed, no community alert brokers were functional.
 
-\begin{document}
+Accordingly, the Rubin project provided a fallback: a simple “alert filtering system” that the project would provide. 
+The AFS would allow Rubin data-rights holders to apply simple user-defined or predefined filters to the Rubin alert stream. 
+This minimized the risk that community alert brokers would not be available, or not provide the functionality required to fulfill users' needs.
 
-% Create the title page.
-\maketitle
-% Frequently for a technote we do not want a title page  uncomment this to remove the title page and changelog.
-% use \mkshorttitle to remove the extra pages
+Today, the situation is quite different. 
+Seven community alert brokers have been approved for direct access to the full Rubin alert stream, with two more planning to operate downstream of a full-stream broker. 
+The brokers have mature systems and active user bases and are operating on live Zwicky Transient Facility alerts. 
+Several brokers have functionality which mirrors that planned for the Rubin AFS. 
+Nevertheless it is reasonable for the Rubin project to ensure that Rubin data-rights holders have access to AFS-like capabilities throughout the survey, even if that service is provided by another entity.
 
-% ADD CONTENT HERE
-% You can also use the \input command to include several content files.
+Therefore Rubin Observatory commits to provide resources to enable the use of the Arizona-NOIRLab Temporal Analysis and Response to Events System (ANTARES),   \citep{2021AJ....161..107M} AFS for Rubin. 
+Rubin will work with CSDC to develop a detailed plan to commission the AFS following the technical description given in the  technical note \citeds{dmtn-226} 
+It is expected the AFS will be commissioned along with the rest of the Rubin system. 
+As such, the AFS should be in place no later than October 1, 2025.
```

Comment:
The AFS  is currently a P2 requirement on DM.  I would prefer to move discussion of a timeline from this memo to [dmtn-226](https://dmtn-226.lsst.io/), which already has a section "timeline". The memo is meant to codify the agreement to collaborate.  I'd prefer to keep all implementation details, including timeline for delivery, in the technical note [DMTN-226](https://dmtn-226.lsst.io/)

Reply:
If we expect the AFS prior to the start of Operations, we need to adjust the DM requirement prioritization

Reply:
I am ok with removing the timeline.